server/rematch/celery.py

```diff
@@ -6,6 +6,8 @@
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'rematch.settings')
+import django
+django.setup()
```

Comment:
I would like to understand the issue here. Is `-A rematch.celery` working for you? Because that's what I'm using.

If it is, I would still like to get the new requirements entry (which I mindlessly forgot adding, sorry for that).

If it isn't, I'd like to understand the issue and wether that's the proper solution.